### PR TITLE
Update location schemas

### PR DESCRIPTION
**Pull request by KacperKluka**
Updated location schema and added the `EnhancedLocationUpdate` schema that's used to send location updates in the Asset Tracking SDK.

### CODE REVIEW

File: test-resources/geo/location-schema.json [hunk 1]
```diff
@@ -74,7 +76,7 @@
           "type": "integer"
         }
       },
-      "additionalProperties": true,
+      "additionalProperties": false,
```
Comment (owenpearson) [inline test-resources/geo/location-schema.json:79]: What's the reason for this change? I originally put this as `false` but it was amended after the discussion [here](https://github.com/ably/ably-asset-tracking-common/pull/1#discussion_r552633168).

Comment (KacperKluka) [reply test-resources/geo/location-schema.json:79]: Good catch, sorry, I wasn't aware of that conversation :sweat_smile: 
I've changed it to `false` because after moving the `altitude` from the `properties` to the `coordinates` I've run tests and expected them to fail but they passed. I thought that's not good and decided to make the schema more strict but given that discussion you've mentioned, I'll change the `additionalProperties` back to `true` :wink:

**Review by owenpearson — APPROVED** [review]
LGTM, thanks
